dgfit/plotting/plot_effsize.py

```diff
@@ -167,10 +178,31 @@ def main():
     ax.plot(component.wavelengths, esize_all, "k-", label="Total")
 
     ax.set_xscale("log")
-    ax.set_xlim(0.1, 1e2)
     ax.set_ylim(0.0, 0.4)
     ax.set_xlabel(r"$\lambda [\mu m]$", fontsize=fontsize)
     ax.set_ylabel(r"average $a [\mu m]$", fontsize=fontsize)
+    ax.set_title("Average a for extinction")
+
+    ax = fax[1, 0]
+    esize = np.empty((component.n_wavelengths, dustmodel.n_components))
+    for k, component in enumerate(dustmodel.components):
+        for i in range(component.n_wavelengths):
+            weights = _emission_all[i, k, :] / _emission_sum_allcomp[i]
+            esize[i, k] = np.sum(_effsize[k, :] * weights) / np.sum(weights)
+
+        ax.plot(component.wavelengths, esize[:, k], symvals[k], label=component.name)
+
+    esize_all = np.empty((component.n_wavelengths))
+    for i in range(component.n_wavelengths):
+        weights = _emission_all[i, :, :] / _emission_sum_allcomp[i]
+        esize_all[i] = np.sum(_effsize * weights) / np.sum(weights)
+    ax.plot(component.wavelengths, esize_all, "k-", label="Total")
+
+    ax.set_xscale("log")
+    ax.set_yscale("log")
+    ax.set_xlabel(r"$\lambda [\mu m]$", fontsize=fontsize)
+    ax.set_ylabel(r"average $a [\mu m]$", fontsize=fontsize)
```

Comment:
It would be good to have a space between "a" and "[um]".   Maybe just move the a outside the math latex?